lms/djangoapps/api_manager/users/tests.py

```diff
@@ -200,6 +200,39 @@ def _create_test_user(self):
         user_id = response.data['id']
         return user_id
 
+    def test_user_is_staff(self):
+        """
+        Test if a user is a staff member
+        """
+        test_uri = self.users_base_uri
+        for i in xrange(1, 7):
+            data = {
+                    'email': 'test{}@example.com'.format(i),
+                    'username': 'test_user{}'.format(i),
+                    'password': 'PassWord1',
+                    'first_name': 'John',
+                    'last_name': 'Doe',
+                    'city': 'Boston',
+                    'is_staff': False,
+            }
+            if i % 2 == 0:
+                data['is_staff'] = True
```

Comment:
`is_staff` assignments can be consolidated into a single line like:

`data['is_staff'] = True if i % 2 == 0 else False`

This would result in more readable test.